src/Tag/Input.php

```diff
@@ -70,7 +70,7 @@ public function open($attr, &$ahtml, &$ihtml)
 		if (isset($properties['FONT-FAMILY'])) {
 			$this->mpdf->SetFont($properties['FONT-FAMILY'], $this->mpdf->FontStyle, 0, false);
 		}
-		if (isset($properties['FONT-SIZE'])) {
+		if (isset($properties['FONT-SIZE']) && $properties['FONT-SIZE'] !== 'auto') {
```

Comment:
I wonder, though, whether we should filter out `font-size: 0` too, as it breaks the input and text-area as well.

Maybe like this:
0 = default, auto = /F 0